app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -42,9 +67,14 @@ public void validate(Object obj, Errors errors) {
             }
         }
         validateEmails(errors, study.getSupportEmail(), "supportEmail");
+        validateEmails(errors, study.getTechnicalEmail(), "technicalEmail");
         validateEmails(errors, study.getConsentNotificationEmail(), "consentNotificationEmail");
     }
     
+    private boolean isInRange(int value, int min) {
+        return (value >= min && value <= 100);
+    }
```

Comment:
Adjusted to use PasswordPolicy.FIXED_MAX_LENGTH for the max length. This could change if we leave Stormpath. Minimum is 2 because that's what Mike asked for when setting password policy. Can't see doing less than 2! And while YML is almost certainly using something longer than 2, I don't know what the length is. We can tighten it later when we find out and have a toggle to change it.